network/network.go

```diff
@@ -1397,19 +1242,32 @@ func (n *network) NodeUptime(subnetID ids.ID) (UptimeResult, error) {
 }
 
 func (n *network) runTimers() {
-	gossipPeerlists := time.NewTicker(n.config.PeerListGossipFreq)
+	pushGossipPeerlists := time.NewTicker(n.config.PeerListGossipFreq)
+	pullGossipPeerlists := time.NewTicker(n.config.PeerListPullGossipFreq)
+	resetValidatorTracker := time.NewTicker(n.config.ValidatorTrackerResetFreq)
 	updateUptimes := time.NewTicker(n.config.UptimeMetricFreq)
 	defer func() {
-		gossipPeerlists.Stop()
+		pushGossipPeerlists.Stop()
+		resetValidatorTracker.Stop()
 		updateUptimes.Stop()
 	}()
 
 	for {
 		select {
 		case <-n.onCloseCtx.Done():
 			return
-		case <-gossipPeerlists.C:
-			n.gossipPeerLists()
+		case <-pushGossipPeerlists.C:
+			n.pushGossipPeerLists()
+		case <-pullGossipPeerlists.C:
+			n.pullGossipPeerLists()
+		case <-resetValidatorTracker.C:
+			if err := n.validatorTracker.ResetBloom(); err != nil {
```

Comment:
I don't see where we update IPs we stored in `validatorTracker.Connected` from `n.peers`?

Reply:
Ahhhh we only update these IPs once we connect to them ourselves. nvm,.